docs/changelog.rst

```diff
@@ -4,6 +4,7 @@ Changelog
 v1.1.1
 ------
 - Allowed MixSourceFolder to mix sources on the fly.
+- Added effects utilities.
```

Comment:
Make this more descriptive. Also add another line about fixing the bug in Ignite.